dotCMS/src/main/java/com/dotcms/rest/api/v1/sites/ruleengine/rules/RuleResource.java

```diff
@@ -77,11 +79,15 @@ protected RuleResource(ApiProvider apiProvider, WebResource webResource) {
     @Path("/rules")
     @NoCache
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-    public Map<String, RestRule> list(@Context HttpServletRequest request, @Context final HttpServletResponse response, @PathParam("siteId") String siteId) {
+    public Map<String, RestRule> list(
+            @Context HttpServletRequest request,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'request' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093298)